core/src/main/java/org/commonjava/maven/ext/core/impl/BaseGroovyManipulator.java

```diff
@@ -158,23 +159,31 @@ void applyGroovyScript( List<Project> projects, Project project, File groovyScri
         {
             try
             {
-                logger.debug ( "Failure when injecting into script {} ", FileUtils.readFileToString( groovyScript, StandardCharsets.UTF_8 ), e );
+                if ( logger.isDebugEnabled() )
+                {
+                    logger.debug( "Failure when injecting into script {}",
+                            FileUtils.readFileToString( groovyScript, StandardCharsets.UTF_8 ), e );
+                }
             }
             catch ( IOException e1 )
             {
-                logger.debug ("Unable to read script file {} for debugging! {} ", groovyScript, e1);
+                logger.debug( "Unable to read script file {} for debugging! {}", groovyScript, e1 );
             }
             throw new ManipulationException( "Unable to inject values into base script", e );
         }
         catch (CompilationFailedException e)
         {
             try
             {
-                logger.debug ( "Failure when parsing script {} ", FileUtils.readFileToString( groovyScript, StandardCharsets.UTF_8 ), e );
+                if ( logger.isDebugEnabled() )
+                {
+                    logger.debug( "Failure when parsing script {}",
+                            FileUtils.readFileToString( groovyScript, StandardCharsets.UTF_8 ), e );
```

Comment:
As per above, I wonder if these actually should be `logger.error` given we are in a failure case (which would make the if block redundant)